### PR TITLE
Fix git conflicts

### PR DESCRIPTION
**Pull request by dongbohu**

### CODE REVIEW

File: .circleci/deploy.sh [hunk 1]
```diff
@@ -85,19 +88,29 @@ done
...
 aws s3 sync submitted_data/ s3://cimr-root/${PR_STR}/ --exclude "request.handled"
 
+# Update the repo
+git pull
+
 # Move submitted YAML files to "processed/" sub-dir
 mkdir -p processed/${PR_STR}/
-git mv -k submitted/*.yml submitted/*.yaml processed/${PR_STR}/
+SUBMITTED_FILES=$(curl -s ${GH_PR_API} | jq -r '.[].filename' | grep "^submitted/") || true
+if [ -z "${SUBMITTED_FILES}" ]; then
```
Comment (ypar) [inline .circleci/deploy.sh:96]: if there are no ${SUBMITTED_FILES}, we shouldn't make processed/${PR_STR}/ dir, no?

File: .circleci/handle_request.sh [hunk 2]
```diff
@@ -60,8 +76,8 @@ INDICATOR_FILENAME="submitted_data/request.handled"
 rm -rf $INDICATOR_FILENAME
 
 # For each yaml file, download and process data
-for f in ${yml_files[*]}; do
-    cimr processor -process -yaml-file $f
+for f in ${SUBMITTED_YAMLS}; do
+    cimr processor -process -yaml-file $f -catalog PR_catalog.txt
```
Comment (ypar) [inline .circleci/handle_request.sh]: -catalog-name PR_catalog.txt

File: .circleci/deploy.sh (hunk 1 above)
Comment (dongbohu) [reply .circleci/deploy.sh:96]: Good point.

File: .circleci/handle_request.sh (hunk 2 above)
Comment (dongbohu) [reply .circleci/handle_request.sh]: How come I used `-catalog foo.txt` and `cimr` doesn't complain?

Comment (ypar) [reply .circleci/handle_request.sh]: you are right, it doesn't! I did change -catalog to -catalog-name at some point but I must have missed an alias somewhere. I will just do some checks and make sure but this is fine.

**Review by ypar — APPROVED** [review]
lgtm. I am approving this PR for merge with minor change of file check -> processed dir mkdir.

File: .circleci/handle_request.sh (hunk 2 above)
Comment (dongbohu) [reply .circleci/handle_request.sh]: I changed it to `-catalog-name` to make the option more consistent.

Comment (ypar) [reply .circleci/handle_request.sh]: sounds good. thank you!

**Pull request merged**